packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -117,3 +117,10 @@
 	overflow: hidden;
 	text-overflow: ellipsis;
 }
+
+.edit-site-global-styles__block-preview-panel {
+	width: 100%;
+	overflow: auto;
+	border: $gray-200 $border-width solid;
+	box-sizing: content-box !important;
```

Comment:
Is the `!important` necessary? It's best to avoid `!important` if possible as it's difficult for plugins to override.

Reply:
Removed the style definition. Created a `scss` variable to manage the height instead.